cli/cmd/lql_show.go

```diff
@@ -46,10 +62,18 @@ func showQuery(_ *cobra.Command, args []string) error {
 	if err != nil {
 		return errors.Wrap(err, "unable to show query")
 	}
+
 	if cli.JSONOutput() {
 		return cli.OutputJSON(queryResponse.Data)
 	}
 
+	if cli.YAMLOutput() {
+		return cli.OutputYAML(&api.NewQuery{
```

Comment:
nit. Were we not able to use `queryResponse.Data` here?  Was this simply an issue of `evaluatorID` being spit out?  If so would adding `omitempty` make sense?